src/api/app/components/add_review_dropdown_component.rb

```diff
@@ -26,7 +25,8 @@ def reviewer_icon_and_text(review:)
   end
 
   def reason_when_review_was_requested(review:)
-    reason = @history_elements.reverse.find { |history_element| history_element.type == 'HistoryElement::RequestReviewAdded' && history_element.description_extension == review.id.to_s }&.comment
+    history_elements = bs_request.history_elements.includes(user)
```

Comment:
https://github.com/openSUSE/open-build-service/wiki/View-Components#avoid-database-queries

Reply:
Argh, there is a query behind the scenes. Thanks for the pointer